src/components/masthead/navToggleButton.tsx

```diff
@@ -22,6 +22,7 @@ class NavToggleButtonBase extends React.Component<Props> {
         className={css(styles.navToggle)}
         onClick={this.props.onClick}
         variant={ButtonVariant.plain}
+        {...{ 'aria-label': 'toggle-navigation-button' }}
```

Comment:
Please see the line I added below. I used the title to provide a descriptive aria label. 

https://github.com/project-koku/koku-ui/blob/master/src/components/masthead/navToggleButton.tsx#L22